src/Uno.UI/Helpers/WinUI/StringUtil.cs

```diff
@@ -0,0 +1,15 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+using System.Text.RegularExpressions;
+
+namespace Uno.UI.Helpers.WinUI
+{
+	internal static class StringUtil
```

Comment:
Maybe some documentation what/why (mainly to discourage people from using it in non-imported code)